tap_s3_csv/sync.py

```diff
@@ -162,6 +171,22 @@ def handle_file(config, s3_path, table_spec, stream, extension, file_handler=Non
     return 0
 
 
+def get_cols_from_metadata(stream):
+    try:
+        mdata = metadata.to_map(stream['metadata'])
```

Comment:
metadata is an array, so it maintains order during serde. So, using column names from metadata